stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/TextFieldUI.kt

```diff
@@ -108,19 +106,12 @@ fun TextFieldSection(
     }
 
     Section(
-        modifier = Modifier.padding(vertical = 8.dp),
+        modifier = modifier,
         title = sectionTitle,
         error = sectionErrorString,
         isSelected = isSelected,
-    ) {
-        TextField(
-            textFieldController = textFieldController,
-            enabled = enabled,
-            imeAction = imeAction,
-            modifier = modifier,
-            onTextStateChanged = onTextStateChanged
-        )
-    }
+        content = content,
+    )
```

Comment:
I initially considered allowing multiple modifiers: one for the `Section` and one for `TextField` but the Jetpack Compose produces a warning if attempting to have modifiers with names other than `modifier`.

I imagine this was done for better practices when writing Composables in order to indicate that a `Composable` should be split up if there are multiple modifiers required in order make it work.

This is mainly why I went with the option to make `TextFieldSection` a `Section` that reads a `textFieldController` for a majority of its values then use `TextField` as part of its content.